src/txs/Tx.tsx

```diff
@@ -103,15 +104,24 @@ function Tx<TxValues>(props: Props<TxValues>) {
   const bankBalance = useBankBalance()
   const { gasPrices } = useTx()
 
-  /* queries: conditional */
-  const shouldTax = !preventTax && getShouldTax(token) && isClassic
+  /* taxes */
+  const preventTaxProp = (preventTax === undefined || !isClassic || preventTax)
```

Comment:
How about to modify the `getShouldTax(token)` to `useShouldTax(token)` in `treasury.ts` like below?
```js
const useShouldTax = (token) => {
  const isClassic = useIsClassic()
  if (!isClassic) return false
  return  isDenomLuna(token) || isDenomTerra(token)
}
```

Reply:
Then we can remove `!isClassic` from this line, it means we don't need to create a new `const preventTaxProp` from `props.preventTax`

Reply:
I implemented your suggestion and turned getShouldTax into a React hook. Since I was having some issues with the call to useIsClassic(), I decided to pass isClassic as a param.